mlrun/api/crud/model_endpoints.py

```diff
@@ -45,22 +45,22 @@
 class ModelEndpoints:
     """Provide different methods for handling model endpoints such as listing, writing and deleting"""
 
-    def create_or_patch(
-        self,
+    @classmethod
```

Comment:
Why is this a `@classmethod`? Note that the crud objects are all singletons, so you have an object to work with - I see no reason for this method to be a class-method, unless I'm missing something. Note that if you need to call other methods, then this doesn't even need to be static - you have `self` to work with.

This comment applies to the rest of the methods created as `@classmethod`.

Reply:
Right, `@classmethod` has been removed
```
class ModelEndpoints:
    """Provide different methods for handling model endpoints such as listing, writing and deleting"""

    def create_model_endpoint(
        self,
        db_session: sqlalchemy.orm.Session,
        access_key: str,
        model_endpoint: mlrun.api.schemas.ModelEndpoint,
    ):
```